### ISSUE TITLE
Enable copying the contents of package manager "preview window" for install/update

### ISSUE THREAD

**Issue by vsfeedback** [NuGet/Home#8324]
See attached.

_This issue has been moved from https://developercommunity.visualstudio.com/content/problem/640739/cant-copy-text-contents-of-the-nuget-package-upgra.html
VSTS ticketId: 945140_
_These are the original issue comments:_

Visual Studio Feedback System on 7/10/2019, 10:32 PM (19 hours ago): <markdown><p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>
</markdown>
_These are the original issue solutions:_
(no solutions)

**Comment by nkolev92** [comment]
Code here, https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PreviewWindow.xaml. 

We are using textblock everywhere but we should use an uneditable textbox instead. 

@rrelyea Probably not a bad new starter issue.

**Comment by zkat** [comment]
I DMed @ghuntley about what the motivating issue was, and from my understanding, it was a matter of sharing some details from the outdated list with a coworker. They do not remember whether it was a snippet of the text, or the entire text, so it's unclear if there's an alternative solution than to make everything copyable a-la-browser.

They also mentioned that they expected the copy shortcut (C-c) to work [the way it works with error messageboxes](https://winaero.com/blog/copy-text-message-box-windows-10/), as an alternative, but that also failed. It's unclear to me whether we should do that in a non-messagebox pane.

Let's continue discussion here and figure out what the best way forward is! I'm also wondering what the actual perf impact of doing the switch to an unediatable textbox is. Will it actually be noticeable? Do we have tools to measure that available to the team?

**Comment by nkolev92** [comment]
There's no automation in our build but you can use https://docs.microsoft.com/en-us/previous-versions/aa969767(v=vs.110). 

From the investigation I did when I created this issue, yes text box is heavier, uses more memory, but it only becomes noticeable if it's used for input (which is not the case for us). 
The dialog is also short lived, so I'd argue there are fewer concerns in our scenario.

The number of textboxes we create also matters. Right now we create a text block for each package entry in the list, we can potentially look into decreasing that if perf is a factor. 

//cc @rrelyea

**Comment by ghuntley** [comment]
Alright I'm back in vswin and this dialog is once again in front of me:

![image](https://user-images.githubusercontent.com/127353/62771432-8f3b9c80-bae0-11e9-9e36-3f1ae29aba17.png)

Thoughts:
- I can't see the contents without expanding it, in the default state it's not great
- If I see something that doesn't look right and want a 2nd opinion from a co-worker then there's no way to select the text within and copy the particular line.
- If I want to keep a textual based work journal or want to share the entire snippet as a gist then there's no way to copy the contents to my clipboard.
- The long established convention of Control + C doesn't work - https://www.raymond.cc/blog/how-to-copy-text-or-error-messages-from-any-dialog-boxes-in-windows/

Other windows in the NuGet manager are individually selectable and can be copied to the clipboard

![image](https://user-images.githubusercontent.com/127353/62771664-16891000-bae1-11e9-8622-8b5e24500c03.png)

**Comment by ghuntley** [comment]
The following text is also not selectable:

![image](https://user-images.githubusercontent.com/127353/62771783-56e88e00-bae1-11e9-84a1-148837bb3446.png)

**Comment by zkat** [comment]
I might be misunderstanding the xaml here, but it looks to me like what we're creating a lot of here is entries in an `ItemsControl` -- are those `TextBlock`s?

Regardless, it doesn't seem to me like the desired outcome is to be able to copy individual lines here, but to be able to select the entire text or otherwise copy it from inside these `ScrollViewer`s (both the update preview and license viewers). I'm inclined to think that just adding a C-c shortcut, and maybe a contextmenu with a copy item, is the best solution in this particular case. How does that sound? Should we bring in a PM for this decision?

**Comment by nkolev92** [comment]
@rrelyea can help. 

Personally I think a C-c shortcut should be just fine.
